src/SFA.DAS.ApplyService.Database/Tables/FinancialReviewClarificationFile.sql

```diff
@@ -0,0 +1,10 @@
+﻿CREATE TABLE [dbo].[FinancialReviewClarificationFile]
+(
+		 [Id] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY CLUSTERED DEFAULT NEWID(),
```

Comment:
Removed **CLUSTERED** so that it matches the other tables

For example:

CREATE TABLE [dbo].[Apply]
(
	[Id] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY DEFAULT NEWID(),